submissions/the-darkesttt/popup/index.html

```diff
@@ -0,0 +1,143 @@
+                            <span>Apps</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/bell.png" alt="">
+                            <span>Bell</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/blogs.png" alt="">
+                            <span>Blogs</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/calendar.png" alt="">
+                            <span>Calendar</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/chat.png" alt="">
+                            <span>Chat</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/classroom.png" alt="">
+                            <span>Classroom</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/collection.png" alt="">
+                            <span>Collection</span>  
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/contacts.png" alt="">
+                            <span>Contacts</span> 
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/documents.png" alt="">
+                            <span>Documents</span> 
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/tanslation.png" alt="">
+                            <span>Translation</span> 
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/files.png" alt="">
+                            <span>Files</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/popup-icons/mail.png" alt="">
+                            <span>Mail</span>
+                        </a>
+                    </li>
+
+                    
+                    <input class="inner-popup_checkbox" type="checkbox" name="" id="inner-popup_checkbox">
```

Comment:
Users must have the ability to open the second part of popup using keyboard. See [video](https://share.vidyard.com/watch/fpeaGmaGWUkyaMU7n31jJ5?) look how focus goes from Mail icon to video recorder panel instead of button more.